sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -122,11 +129,8 @@ export interface ReceivedMessageWithLock extends ReceivedMessage {
 
 // @public
 export interface Receiver<ReceivedMessageT> {
+    browseMessages(options?: BrowseMessagesOptions): Promise<ReceivedMessage[]>;
```

Comment:
@bterlson, `browse` or `browseMessages` ?
The team is leaning towards the latter.
The one problem it brings is that the default value is 1, but the name suggests there is more than 1 :)

Reply:
On the other hand if we are going with `browseMessages`, then shouldnt `receiveBatch` be `receiveMessages`?

Reply:
Am fine with `browseMessages`, though I feel like messages is somewhat the point of all of this so can be implicit.